### ISSUE TITLE
Compatibility with sbt auto-testing

### ISSUE THREAD

**Issue by bijoutrouvaille** [scallop/scallop#67]
Hi,

I'm trying out this excellent library, but I'm experiencing a certain frustration, which I can not seem to eliminate. When running sbt I use the ~ operator to automatically execute unit tests when a change is detected, but if Scallop detects an invalid usage, it causes sbt to exit entirely, thus eliminating the ability to auto-test.

Do you have any suggestions to ameliorate this situation, or is this currently a limitation?

**Comment by Rogach** [comment]
In scallop test suite, it is done [via setting an internal flag](https://github.com/Rogach/scallop/blob/master/src/test/scala/ConfTest.scala#L9). Unfortunately, currently that flag is marked private - I'll fix it now, and publish a snapshot.

**Comment by Rogach** [comment]
I published a snapshot (for scala 2.10.x):

``` scala
resolvers += "sonatype snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
libraryDependencies += "org.rogach" %% "scallop" % "0.9.2~6~SNAPSHOT"
```

**Comment by bijoutrouvaille** [comment]
Thanks for the quick attention! 
After the update, I just inserted `org.rogach.scallop.throwError.value = true` at the top of my suite and the issue seems to have been dealt with.

**Comment by Rogach** [comment]
Great! I'll publish non-snapshot version shortly, then.

**Comment by Rogach** [comment]
I published version `0.9.3`, it should be available from Maven Central in 1-2 hours.
